components/vc/pdu/src/HW/HW_adc_componentSpecific.c

```diff
@@ -29,12 +29,29 @@
 #define ADC_PRECALIBRATION_DELAY_ADCCLOCKCYCLES 2U
 #define ADC_CALIBRATION_TIMEOUT                 10U
 
+#define ADC_BANK1_CHANNEL_MUX_LP1_SNS       ADC_CHANNEL_7
+#define ADC_BANK1_CHANNEL_MUX_LP2_SNS       ADC_CHANNEL_10
+#define ADC_BANK1_CHANNEL_MUX_LP3_SNS       ADC_CHANNEL_11
+#define ADC_BANK1_CHANNEL_MUX_LP4_SNS       ADC_CHANNEL_12
+#define ADC_BANK1_CHANNEL_UVL_BATT          ADC_CHANNEL_1
+#define ADC_BANK1_CHANNEL_5V_VOLTAGE        ADC_CHANNEL_4
+#define ADC_BANK1_CHANNEL_MCU_TEMP          ADC_CHANNEL_TEMPSENSOR
+
+#define ADC_BANK1_CHANNEL_MUX_LP5_SNS       ADC_CHANNEL_13
+#define ADC_BANK2_CHANNEL_MUX_LP6_SNS       ADC_CHANNEL_14
+#define ADC_BANK2_CHANNEL_MUX_LP7_SNS       ADC_CHANNEL_15
+#define ADC_BANK2_CHANNEL_MUX_LP8_SNS       ADC_CHANNEL_8
+#define ADC_BANK2_CHANNEL_MUX_LP9_SNS       ADC_CHANNEL_9
+#define ADC_BANK2_CHANNEL_MUX2_THERMISTORS  ADC_CHANNEL_2
+#define ADC_BANK2_CHANNEL_MUX2_HP_CS        ADC_CHANNEL_3
+
 /******************************************************************************
  *                              E X T E R N S
  ******************************************************************************/
 
 extern ADC_HandleTypeDef hadc1;
 extern ADC_HandleTypeDef hadc2;
+extern DAC_HandleTypeDef hdac;

```

Comment:
If youre using this youll need a few more things. Youll need to select dac in the includes/hw/yaml, youll need to include the header, etc... and it should all live in a dac file, not in adc